src/encoder/encodervorbis.cpp

```diff
@@ -37,39 +37,15 @@ A lot of stuff has been stolen from:
 
 EncoderVorbis::EncoderVorbis(EncoderCallback* pCallback)
         : m_bStreamInitialized(false),
+          m_vblock({}),
```

Comment:
IMHO all members are initialized before the first usage. If this is wrong there is a real issue. 

> The suggestion would be to remove that particular warning from GCC 

If it is possible to remove the warning for this particular file only, it will work for me.
But the design issue and the redundant initialize issue remains.
I would prefer to fix this, but that's more (unimportant) work.